### PR TITLE
Management Transfer

### PR DESCRIPTION
**Pull request by andygray**
``` 
 /**
   * @dev Allow management to transfer tokens. Will be used to transfer unsold tokens.
   * @param _beneficiary Token beneficiary
   * @param _tokenAmount Amount of tokens to deliver
   */
  function managementTransfer(address _beneficiary, uint256 _tokenAmount) external onlyManagement {
    _deliverTokens(_beneficiary, _tokenAmount);
    emit ManagementTransfer(msg.sender, _beneficiary, _tokenAmount);
  }
```

 * [x] New function `managementTransfer`
 * [x] Removed unused `getNow`
 * [x] New tests

### CODE REVIEW

File: contracts/pixie/PixieCrowdsale.sol [hunk 1]
```diff
@@ -11,6 +11,8 @@ contract PixieCrowdsale is Crowdsale, Pausable {
 
   event MinimumContributionUpdated(uint256 _minimumContribution);
 
+  event ManagementTransfer(address indexed caller, address indexed beneficiary, uint256 amount);
```
Comment (jamesmorgan) [inline contracts/pixie/PixieCrowdsale.sol]: I would change 'amount'  to 'tokens' so it matches the parent event naming convention

File: test/truffle/pixie/PixieCrowdsale.test.js [hunk 2]
```diff
@@ -950,7 +949,7 @@ contract('PixieCrowdsale', function ([owner, investor, wallet, purchaser, author
     describe('pre sale rate', async function () {
 
       beforeEach(async function () {
-        await increaseTimeTo(this.preSaleCloseTime - duration.seconds(10)); // set time to just before pre sale close
+        await increaseTimeTo(this.afterClosingTime - duration.seconds(10)); // set time to just before pre sale close
```
Comment (jamesmorgan) [inline test/truffle/pixie/PixieCrowdsale.test.js]: Invalid comment

Comment (jamesmorgan) [reply test/truffle/pixie/PixieCrowdsale.test.js]: Why is this needed, test describe block is for presale

File: test/truffle/pixie/PixieCrowdsale.test.js [hunk 3]
```diff
@@ -971,4 +970,80 @@ contract('PixieCrowdsale', function ([owner, investor, wallet, purchaser, author
 
   });
 
+  describe('Management transfer - allows management to transfer tokens (at their discretion)', function () {
+
+    beforeEach(async function () {
+      await this.crowdsale.addToWhitelist(authorized);
+      await this.crowdsale.addToManagementWhitelist(authorized);
```
Comment (jamesmorgan) [inline test/truffle/pixie/PixieCrowdsale.test.js]: Why add to both?

File: test/truffle/pixie/PixieCrowdsale.test.js [hunk 4]
```diff
@@ -971,4 +970,80 @@ contract('PixieCrowdsale', function ([owner, investor, wallet, purchaser, author
...
+      });
+    });
+
+
+    describe('during crowdsale make transfer to address', async function () {
+
+      const LOTS_OF_TOKENS = 12234234234234;
+
+      beforeEach(async function () {
+        await increaseTimeTo(this.openingTime - duration.seconds(10)); // set time to after ICO ends
```
Comment (jamesmorgan) [inline test/truffle/pixie/PixieCrowdsale.test.js]: Invalid comment (I think), tast says during, this caunges to before ico?

File: test/truffle/pixie/PixieCrowdsale.test.js [hunk 5]
```diff
@@ -971,4 +970,80 @@ contract('PixieCrowdsale', function ([owner, investor, wallet, purchaser, author
...
+      });
+
+      it('should assign tokens after closing time', async function () {
```
Comment (jamesmorgan) [inline test/truffle/pixie/PixieCrowdsale.test.js]: Invalid it() I think

File: test/truffle/pixie/PixieCrowdsale.test.js (hunk 3 above)
Comment (andygray) [reply test/truffle/pixie/PixieCrowdsale.test.js]: You need to be whitelisted to send. The weird thing is I thought it was at the top. Have a look

File: test/truffle/pixie/PixieCrowdsale.test.js (hunk 4 above)
Comment (andygray) [reply test/truffle/pixie/PixieCrowdsale.test.js]: Drop time - not needed.

**Pull request merged**